src/TextInput/index.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+
+import * as React from 'react';
+import { TextInput as RNTextInput, View } from 'react-native';
+
+import { StyleSheet } from '..';
+import FormLabel from './FormLabel';
+
+type Props = {
+  +size?: 'small' | 'normal',
+  +placeholder?: string,
+  +value?: string,
+  +disabled?: boolean,
+  +onFocus?: () => void | Promise<any>,
+  +onBlur?: () => void | Promise<any>,
+  required?: boolean,
+  +label: string,
+};
+
+type State = {
+  isFocused: boolean,
+};
+
+const colors = {
+  colorTextInputDisabled: '#bac7d5',
+  colorTextInput: '#46515e',
+  borderColorInput: '#bac7d5',
+  colorPlaceholderInput: '#bac7d5',
+  borderColorInputFocus: '#0176D2',
+};
+
+const getFontSize = size => (size === 'small' ? 14 : 14);
```

Comment:
why is it the same if size is small, or not small?

Reply:
Just take these values from https://github.com/kiwicom/orbit-components

Reply:
I think we should rather use these `import { defaultTokens } from '@kiwicom/mobile-orbit';`

Reply:
https://github.com/kiwicom/orbit-design-tokens/blob/4930822a7ae11dca6026d5845d37d6eb65b00907/output/theo-spec.common.js#L27

https://github.com/kiwicom/orbit-design-tokens/blob/4930822a7ae11dca6026d5845d37d6eb65b00907/output/theo-spec.common.js#L166

Reply:
That is ok, but then is function is useless, just set fontSize to 14